mlperf_logging/rcp_checker/rcp_checker.py

```diff
@@ -292,12 +360,14 @@ def _create_interp_rcp(self, benchmark, target_bs, low_rcp, high_rcp):
                          'Epochs to converge': [],
                          'RCP Mean': mean,
                          'RCP Stdev': stdev,
-                         'Max Speedup': mean / min_epochs}
-        logging.info(" Creating interpolation record")
-        logging.info(" Low RCP: %s", low_rcp)
-        logging.info(" High RCP: %s", high_rcp)
-        logging.info(" Intepolation record: %s", interp_record)
-        self.rcp_data[interp_record_name] = interp_record
+                         'Max Speedup': mean / min_epochs,
+                         'Min Epochs': min_epochs}
+        if show_rec:
+            logging.info(" Creating interpolation record")
```

Comment:
`logging` can already control what to print out depending on the verbose level, so `show_rec` and `if` is a little redundant here. I would just, for example, add a `argparse` flag called `verbose_level` or something, and make the printing here to be `logging.warning()`

Reply:
OK, I am using the verbose param from args.